### ISSUE TITLE
This player is powered by pisiganage.com ticker is coming

### ISSUE THREAD

**Issue by SarathKumarMB1833** [colloqi/piSignage#278]
i am using the palyer only open source licence for 2 free player for a local host and when i load the content the ticker is coming.How can i solve this

**Comment by hitesh83** [comment]
Register your players ID on pisignage website in your account.
Download Licence txt files.
Upload them in open source server, keep username in settings same as your pisignage account.

**Comment by AbhijithBn** [comment]
Hi @SarathKumarMB1833  

Did you check the points mentioned in this article https://help.pisignage.com/hc/en-us/articles/360001256851-This-player-is-powered-by-pisignage-com?

Especially,

- license should be generated as "Player only" while registering
- installation name should be same as your username at [pisignage.com](http://pisignage.com/) (not email ID)
- license file name should be same and without any extra characters like ~1
- License file is uploaded to your server and player is online

**Comment by DorkSaber** [comment]
@AbhijithBn Yes I second @frankchau93, I'm accessing my open source server via a Cloudflare reverse proxy, and I'm also getting domain not matching. How do I resolve this? Is there somewhere in the installation I can input the reverse proxy domain? I really don't want to use port forwarding. And even if I did, would that even work? Or do I have to have them on the same lan? There must be a way. @frankchau93 did you figure it out?

**Comment by hitesh83** [comment]
the domain here is not referred to your reverse proxy, it might be referring to **UserID**.pisigange.com, this **UserID** need to mentioned in your opensource server's setting.
![image](https://github.com/user-attachments/assets/ed5f3884-1f10-46da-9052-2cd4e1889488)
